src/components/FormBook.js

```diff
@@ -0,0 +1,13 @@
+const FormBook = () => (
+  <form>
+    <h1>ADD NEW BOOK</h1>
+    <input type="text" placeholder="Book Title" />
+    <select placeholder="Category">
+      <option>Example 1</option>
+      <option>Example 2</option>
+      <option>Example 3</option>
+    </select>
+  </form>
```

Comment:
- [ ] This component should render a form with inputs for both the title and author of the book. You're currently having only the title input. Add the author input to your form.